Source/WebCore/platform/network/OrganizationStorageAccessPromptQuirk.h

```diff
@@ -33,13 +33,15 @@ namespace WebCore {
 
 struct OrganizationStorageAccessPromptQuirk {
     String organizationName;
-    HashMap<RegistrableDomain, Vector<RegistrableDomain>> domainPairings;
+    HashMap<RegistrableDomain, Vector<RegistrableDomain>> quirkDomains;
+    Vector<URL> triggerPages;
 
     bool isHashTableDeletedValue() const { return organizationName.isHashTableDeletedValue(); }
 
-    OrganizationStorageAccessPromptQuirk(String&& organizationName, HashMap<RegistrableDomain, Vector<RegistrableDomain>>&& domainPairings)
+    OrganizationStorageAccessPromptQuirk(String&& organizationName, HashMap<RegistrableDomain, Vector<RegistrableDomain>>&& quirkDomains, Vector<URL> triggerPages)
```

Comment:
Nit - `&&` to signify the change in ownership.

```suggestion
    OrganizationStorageAccessPromptQuirk(String&& organizationName, HashMap<RegistrableDomain, Vector<RegistrableDomain>>&& quirkDomains, Vector<URL>&& triggerPages)
```

Reply:
Oh, good catch